app/views/feeds/_form_auth_tokens.html.erb

```diff
@@ -1,7 +1,19 @@
-<% if feed.tokens.present? %>
-  <%= form.fields_for :feed_tokens, feed.tokens do |fields| %>
-    <%= render "form_auth_token", form: fields %>
+<% feed.tokens.build %>
+
+<%= form.fields_for :feed_tokens do |fields| %>
+  <%= turbo_frame_tag "feed-token-#{fields.object.id}" do %>
+    <% if fields.object.marked_for_destruction? %>
+      <%= fields.hidden_field :_destroy, value: "1" %>
+    <% elsif fields.object.id.blank? %>
```

Comment:
for the token that is built, i decided to hide it until the Add Token checkbox is checked.

Reply:
This does feel a bit awkward to add to the "toggle field controller".  Though I can see how similar it is.

I'd suggest for fast-feedback, you just always show the new token "label" field, and hide the new token "token" field.  Once the user adds a label and saves, they create a token.

Longer term, this should probably be an "add token" button.  That (ideally) allows you to add as many tokens as you want.  But - let's get some quick feedback first.